cmd/tailscaled/tailscaled.go

```diff
@@ -276,29 +276,13 @@ func main() {
 	}
 
 	if buildfeatures.HasTPM {
+		var err error
```

Comment:
I want this part up in `main` to read like only 3l lines:

    if buildfeatures.HasTPM {
          handleTPMFlags()
    }

Then rest in func or funcs below.